### DISCUSSION TITLE
Page Block System

### DISCUSSION THREAD

**Discussion by PepijnVDK1** [Laravel-Backpack/community-forum#815] [Q&A (Help)]
Hi, 
I recently bought the pro version and I would love to build a page block system where we have a select box with page-block types:

- photo-text-block
- text-block
- ……

Fields should hide/show depending on which select-value is selected.
Somewhat the same as how the pageManager add-on module is built.

This is fairly easy to do with CrudField Javascript. Yet there is something I'm stuck with. 
I need the above functionality in a repeater field. So that the user can create as many page-blocks as needed and reorder them etc…

Alternatively it would be nice to do this in a separate relational resource. Therefor I was wondering if it's possible, when going to edit a page, if there is a way to show a table with all related page blocks and perhaps create them from that view?

/page/{id}/block/{id}/edit

I've tried searching this throughout the docs and these discussions but was not able to find an answer.

**Comment by karandatwani92** [comment]
Hey @PepijnVDK1 

To build something like this `/page/{id}/block/{id}/edit` having separate relational resource,
Check https://backpackforlaravel.com/articles/tutorials/nested-resources-in-backpack-crud

Let me know if this helps
